frontend/packages/core/src/Input/stories/toggle-button-group.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import type { Meta } from "@storybook/react";
+
+import ToggleButtonGroup from "../toggle-button-group";
+
+export default {
+  title: "Core/Input/ToggleButtonGroup",
+  component: ToggleButtonGroup,
+} as Meta;
+
+export const ExclusiveDemo = () => {
```

Comment:
whats the difference between a stories file and a regular file?

Reply:
when the files have `.stories` in the name storybook picks it up and displays the exports in the page